code-challenges.test.js

```diff
@@ -14,39 +14,122 @@
+    expect(iDontWantItUnless(fruitArray, letterA)).toEqual(["Mango", "Apricot", "Peach"])
+    expect(iDontWantItUnless(fruitArray, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+    })
+})
 
-const letterA = "a"
-// Expected output: ["Mango", "Apricot", "Peach"]
-const letterE = "e"
-// Expected output: ["Cherry", "Blueberry", "Peach"]
+//output: ReferenceError: iDontWantItUnless is not defined - good error
 
 // b) Create the function that makes the test pass.
 
+//make function that seeks out words that have a specific letter
+//name the function iDontWantItUnless and give it to parameters
+// use .filter to citerate through given array and create a new array where the wanted word can be pushed into
+// use .tolowercase to catch the capitol characters and use include to get the second parameter
+// call on the newArray
+
+const iDontWantItUnless = (array, letter) =>{
+    let newArray = array.filter((word) => {
+    return word.toLowerCase().includes(letter)
+    })
+    return newArray
+ }
+
+  //output: pass
+
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
 
+//Elyse i freaking love this problem, in the near future i want to create a card game for my portfolio 
+
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
-// Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
-// Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
-const hand4 = [7, 2, 7, 2, 7]
-// Expected output: true
+// create a test function named myFavoriteHandInPoker
+
+describe("myFavoriteHandInPoker", () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    // Expected output: true
+    const hand2 = [5, 5, 3, 3, 4]
+    // Expected output: false
+    const hand3 = [5, 5, 5, 5, 4]
+    // Expected output: false
+    const hand4 = [7, 2, 7, 2, 7]
+    // Expected output: true
+    it("takes in an array of 5 numbers and determines whether or not the array is a full house", () => {
+    expect(myFavoriteHandInPoker(hand1)).toEqual(true)
+    expect(myFavoriteHandInPoker(hand2)).toEqual(false)
+    expect(myFavoriteHandInPoker(hand3)).toEqual(false)
+    expect(myFavoriteHandInPoker(hand4)).toEqual(true)
+    })
+ })
+
+//output: ReferenceError: myFavoriteHandInPoker is not defined -good error
 
 // b) Create the function that makes the test pass.
+
+//make function that will check for triples and doubles 
+//not really sure how to do this but solving it in my head,
+//i would make it iterate through with .length of array to stop, 
+//while it iterates through the array check for nummbers 2 - 10 and if it hits I need it to store it somewhere and I would need one number to show up twice and another to show up 3 or I could use the filter method to pluck the numbers out and see if there is one double pair && a trip pair inside of the array 
```

Comment:
Your logic seems to be heading in the right direction.  I do need to see an attempt for your code on this problem.  Try writing the code for what you wrote here and see what happens.